addons/google_calendar/models/google_sync.py

```diff
@@ -151,6 +151,8 @@ def _sync_odoo2google(self, google_service: GoogleCalendarService):
                 if record.google_id and record.need_sync:
                     record.with_user(record._get_event_user())._google_delete(google_service, record.google_id)
             for record in new_records:
+                if record._is_google_insertion_blocked(sender_user=self.env.user):
+                    continue
```

Comment:
The cod echanged a lot since I checked. I thought the insertion would be made with a with_user to make sure the syncing user would be used :thinking: I am not sure to understand when these issues arise.